### PR TITLE
fix sort on single element issue

### PR DESCRIPTION
**Pull request by liorblob**
if a single element is returned, convert it to array.
Solves https://github.com/demisto/etc/issues/10861

### CODE REVIEW

File: Integrations/integration-AnomaliThreatStream.yml [hunk 1]
```diff
@@ -523,3 +527,4 @@ script:
       type: number
     description: Checks the reputation of the given IP
 hidden: false
+releaseNotes: "Fix parse result sort issue"
```
Comment (anara123) [inline Integrations/integration-AnomaliThreatStream.yml]: release notes are unnecessary here. `releaseNotes: "-"`

File: Integrations/integration-Remedy_AR.yml [hunk 2]
```diff
@@ -181,3 +181,4 @@ script:
       type: string
     description: Retrieves server details
 hidden: false
+releaseNotes: "-"
```
Comment (anara123) [inline Integrations/integration-Remedy_AR.yml]: no need. remove this change and pull from master

**Comment by anara123** [comment]
@liorblob open new issue for documentation and put here a link for that issue

**Comment by liorblob** [comment]
Documentation: https://github.com/demisto/etc/issues/11057
Need to add outputs to the commands

**Comment by anara123** [comment]
@liorblob You mean some commands missing outputs?

File: Integrations/integration-AnomaliThreatStream.yml [hunk 3]
```diff
@@ -266,6 +266,10 @@ script:
     }
 
     var parseResult = function(command, result) {
+        if(!(result['objects'] instanceof Array)){
```
Comment (asafshen) [inline Integrations/integration-AnomaliThreatStream.yml]: what about `parseReputationResult`?
```
var objects = dq(result, 'objects');
objects.sort(function(x, y) {return x.created_ts < y.created_ts});
```

also - I think we should handle a case that the return value is not  an object, and not trust the api to return an object (`res.Body`)

Comment (liorblob) [reply Integrations/integration-AnomaliThreatStream.yml]: `parseReputationResult` is always called from `parseResult`, the bug was actually there but the source is from `parseResult`. And I didn't fully understand - did you mean the `objects` field in the result not being returned at all or the result itself not being an object?

Comment (asafshen) [reply Integrations/integration-AnomaliThreatStream.yml]: thanks for the `parseReputationResult` clarification

regrading the second thing - if `result` (which is the body of the returned request) is not an object - we will also get an exception here

Comment (liorblob) [reply Integrations/integration-AnomaliThreatStream.yml]: Added checks for `result`

**Pull request merged**